digitalpy/core/zmanager/impl/zeroless_pusher.py

```diff
@@ -20,10 +18,10 @@ def subject_bind(self, address: int, port: str):
         """create the ZMQ zocket
         """
         self.pusher_client.connect(address, port)
-        self.push = self.pusher_client.push()
         
-    def subject_send_request(self, request: Request):
+    def subject_send_request(self, request: Request, protocol: str):
         """send the message to a Puller
         """
-        serialized_request = self.pusher_formatter.serialize(request)
-        self.push(serialized_request)
\ No newline at end of file
+        push = self.pusher_client.push()
+        self.pusher_formatter.serialize(request)
+        push(b",".join([request.get_sender().encode(), request.get_context().encode(), request.get_action().encode(), request.get_format().encode(), protocol.encode(), request.get_values()]))
```

Comment:
**[pep8]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>W292 no newline at end of file